apps/projects/tasks.py

```diff
@@ -1,9 +1,9 @@
 import importlib
 
-from background_task import background
+from celery import shared_task
 
 
-@background(schedule=1)
+@shared_task
 def send_async_no_object(email_module_name, email_class_name, object, args, kwargs):
```

Comment:
can we rename object to object_dict or use typing hint? @goapunk I think we should do that while we touch legacy code, is fine if only some parts has it, it will grow slowly. Already talked about it breifly with @hklarner